app/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,6 @@
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="connection_unavailable_title">Sorry! Network Connection Unavailable</string>
     <string name="connectivity_error_message">There was a connection error</string>
+    <string name="toast_refreshing_data">Refreshing Data</string>
```

Comment:
name it something like "refresh_data_message" as string resource names shouldn't contain implementation details such as "toast"